views/partials/header.html

```diff
@@ -5,7 +5,7 @@
 			<div class="o-header__top-wrapper">
 				<div class="o-header__top-column o-header__top-column--left">
 					{{#if enableSidebarMenu}}
-						<a class="o-header__top-link o-header__top-link--menu" aria-controls="o-header-drawer" role="button" aria-expanded="false" on='tap:sidebar.open' data-vars-link-type="sidebar-menu-open" data-vars-link-text="Menu">
+						<a href="#" class="o-header__top-link o-header__top-link--menu" aria-controls="o-header-drawer" role="button" aria-expanded="false" on='tap:sidebar.open' data-vars-link-type="sidebar-menu-open" data-vars-link-text="Menu">
```

Comment:
I have little AMP knowledge and I know something are just "THIS IS HOW IT MUST BE"

**Suggestion/Comment:** Considering this doesn't link any where should we be using an anchor?

Reply:
I like how the BBC(?) does it where if JavaScript hasnt loaded the menu is an anchor jump to the bottom of the page.

I am mirroring the live FT.com site which does this in this change, I do not like it but want to improve things in a low risk way if possible.
Note that the anchor does have `role=button` to give it semantics which better match its affordance.

Reply:
Actually, FT.com *does* work that way, and is likely why they didnt use a button in the first place. Mystery solved.

Reply:
Going to see if I can make this better...

Reply:
> I like how the BBC(?) does it where if JavaScript hasnt loaded the menu is an anchor jump to the bottom of the page.

FT.com does this too :) 

> Note that the anchor does have role=button to give it semantics which better match its affordance.

Thanks for making me aware of that, I forgot to check that

Reply:
Like you say, this is better than whats there already so thats good 😄

Reply:
Async replies threads FTW 😂

Reply:
I have made this work how FT.com works by making sure it links to the sidebar when it is not enhanced with JavaScript.

Reply:
Nice one 😄  thanks Nick 🎉 

As I say I am not very familiar with the quirks of amp so I will trust that you test this and its all good

Reply:
Will test this in a few mobile browsers to double check it's still good.